lava_test_plans/testcases/android-cts.yaml

```diff
@@ -0,0 +1,11 @@
+{% extends "testcases/master/template-android-xts.jinja2" %}
+
+{% set xts_test_url = TEST_CTS_URL %}
+{% set xts_version = TEST_CTS_VERSION %}
+
+{% set xts_test_params = xts_test_params|default("cts") %}
+{% set test_name = test_name|default("cts") %}
+{% set xts_test_suite_name = xts_test_suite_name|default("cts-lkft") %}
+{% set xts_test_path = xts_test_path|default("android-cts") %}
+{% set xts_pkg_name = xts_pkg_name|default("android-cts.zip") %}
+{% set xts_metadata_prefix = xts_metadata_prefix|default("cts") %}
```

Comment:
please move this file to lava_test_plans/testcases/master/template-android-cts.jinja2
and make it a template.